modules/ROOT/pages/view-graphql-api-schemas-and-run-queries.adoc

```diff
@@ -0,0 +1,63 @@
+= View GraphQL API Schemas and Run Queries
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+ Use the GraphQL console to explore and understand the available GraphQL schemas, write, run, and troubleshoot GraphQL queries. The console is an easy-to-use interface for API app developers to interact with GraphQL APIs in Exchange. GraphQL APIs are schema definitions that contain the object types and definitions that are used to interact with the API through the GraphQL specification. The GraphQL console enables app developers to query an API that supports this language in a more flexible way than REST. The console supports standard GraphQL query syntax. The console is located in the asset details page.
+
+image::ex-graphql-console.png[GraphQL console in the asset details page]
+
+[calloutlist]
+. *Schema*: Use the Schema explorer to explore the GraphQL API schema types and documentation.
+. *Query*: Use the Query editor to write and run queries.
++
+* *Variables*: Use this tab to enter variables that you want to use to query.
+* *Headers*: Use this tab to enter header information that you want to use to query.
+. *Results*: Use this panel to review and troubleshoot queries.
+
+
+== Before You Begin
+
+To use the GraphQL console, the Exchange Viewer permission is required. For more information, see xref:to-set-permissions.adoc[Set User Access Permissions].
+
+== View GraphQL API Schemas
+
+Explore the schema by inspecting the types, fields, and relationships defined in the schema to see a visual representation of the data and operations. Use the Schema explorer to inspect the query methods that are available in the API schema. The documentation explorer renders the information and allows the ability to navigate through the documentation of the API.
+
+Use the *Search* box or keyboard shortcuts to search the schema documentation to find relevant information about specific types, fields, or operations. The search results highlight the search criteria in the corresponding sections of the documentation.
+
+. In the Exchange landing page, click the *Any type* filter and select *GraphQL APIs* from the list.
+. Select a GraphQL API from the results.
+. In the navigation bar, click *Console*.
+. In the *Schema* panel, review the schema documentation.
+. Click *ExchangeGraph* to see the root schema types for each operation.
+. Search the schema documentation using one of the following methods: 
+.. Use the *Search for content* box by clicking the three dots icon, selecting and entering a keyword in the *Search for content* box.
+.. Use the *CTRL+F* or *Command+F* keyboard shortcuts by pressing a keyboard shortcut combination and entering a keyword in the *Search* box.
```

Comment:
Same here, one line says `Search box` and the other says `Search for content box`.